source/numerics.tex

```diff
@@ -6818,7 +6818,7 @@
 elements of \tcode{v}.\footnote{This copy constructor creates
 a distinct array rather than an alias.
 Implementations in which arrays share storage are permitted, but they
-shall implement a copy-on-reference mechanism to ensure that arrays are
+would need to implement a copy-on-reference mechanism to ensure that arrays are
```

Comment:
I'm going to merge as-is, but I wonder if we should just drop this -- in modern C++ this would also require taking a lock to provide thread-safety for const operations, which seems like a silly thing to bother describing as "permitted".